src/frontend/src/layouts/PageTitle/PageTitle.tsx

```diff
@@ -40,12 +41,13 @@ const PageTitle: React.FC<PageTitleProps> = ({ title, previousPages, headerRight
         bgcolor={theme.palette.background.default}
       >
         <Grid container>
-          <Grid item xs={6} md={4}>
+          <Grid item xs={6} md={tabs ? 4 : 8} display="flex" alignItems={'center'}>
```

Comment:
On small (especially mobile) screens, the title becomes so stretched that it's one letter per line